package.json

```diff
@@ -16,6 +16,13 @@
     "hammerjs": "^2.0.8"
   },
   "devDependencies": {
+    "babel-core": "^6.0.0",
```

Comment:
Were the babel dependencies added to accommodate vtk.js? Is its dist package not ES5? @jourdain

Reply:
We use preset-env and target only the last 2 version of browsers. So there is some ES6 left that don't get transpiled such as arrow function depending on browser support.

Reply:
I mean I don't know about that project, I was talking about vtk.js.

Reply:
Thanks, that's what I was asking about